### PR TITLE
[refactor](cache) remove entrance of partition cache

### PR DESCRIPTION
**Pull request by 924060929**
the partition cache has many bugs and few user use it, so we remove the entrance of partition cache, and we will refactor it until support mtmv rewrite function

### CODE REVIEW

**Comment by 924060929** [comment]
run buildall

**Comment by 924060929** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheAnalyzer.java [hunk 1]
```diff
@@ -142,11 +142,12 @@ private void checkCacheConfig() {
...
-            if (context.getSessionVariable().isEnablePartitionCache()) {
-                enablePartitionCache = true;
-            }
-        }
+        enablePartitionCache = false;
+        // if (Config.cache_enable_partition_mode) {
+        //     if (context.getSessionVariable().isEnablePartitionCache()) {
+        //         enablePartitionCache = true;
+        //     }
+        // }
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheAnalyzer.java:150]: remove useless code, not comment them?

Comment (924060929) [reply fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheAnalyzer.java:150]: > remove useless code, not comment them?

I think the readers can use git blame to find this pr

Comment (924060929) [reply fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheAnalyzer.java:150]: maybe we will revert this comment?

**Comment by 924060929** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**